.gitignore

```diff
@@ -2,6 +2,4 @@
 *.swp
 *.pcm
 *.bkp
-test*root
```

Comment:
What happened to these lines?

Reply:
I only remember test*root. There is a committed file on git with that name, and I wasn't able to get all other files until that issue was solved, so I removed that requirement.